### DISCUSSION TITLE
Overriding Theme Styles

### DISCUSSION THREAD

**Discussion by MatthewKyle22** [ApollosProject/apollos-templates#349] [Q&A] [answered]
Hey there!

I'm currently working to make our dark theme version look better but am having trouble overriding specific component styles.

As you can see in my attached photo, the ActionBarItem is a gray color which looks fine in light mode, but when switched to dark mode, it easily gets overlooked.

Ideally I'd like to override the ActionBarItem's fill for the icon and color for the label.
What might be the best way of doing this?

Thanks in advance!
![Screen Shot 2021-05-07 at 9 17 01 AM](https://user-images.githubusercontent.com/11671126/117469495-6cd7ec80-af1b-11eb-8c2d-674633782e9a.png)

**Comment by MatthewKyle22** [comment, marked as answer]
Found out there is a tint property for this particular component that I can pass a string that will work for what I'm wanting.

**Reply by redreceipt** [reply]
Excellent. thanks for posting a discussion. I'll mark this solved.
